### PR TITLE
AQT doc devices and getting started tutorial

### PR DESCRIPTION
**Pull request by alfrisch**
This PR adds AQT documentation and contains a `*.md` describing how to retreive tokens and URLs of AQT devices and a Jupyter notebook on a simple getting started example which shows how to execute a basic quantum circuit via the AQT API.

### CODE REVIEW

File: docs/tutorials/aqt/getting_started.ipynb [hunk 1]
```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "0341a89e3dc0"
+   },
+   "source": [
+    "##### Copyright 2020 Alpine Quantum Technologies GmbH"
```
Comment (balopat) [inline docs/tutorials/aqt/getting_started.ipynb:9]: This needs to be:

```suggestion
    "##### Copyright 2020 The Cirq Developers"
```

AQT is part of The Cirq Developers automatically being a contributor to the repo.

File: docs/tutorials/aqt/getting_started.ipynb [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "576763bf5e42"
+   },
+   "source": [
+    "## Sample a quantum circuit\n",
+    "\n",
+    "You can then use that device in Cirq. For example, preparing the Bell state\n",
+    "\n",
+    "\\\\[ |\\psi\\rangle=\\frac{1}{\\sqrt{2}}(|00\\rangle-\\mathrm{i}|11\\rangle) \\\\]\n",
```
Comment (balopat) [inline docs/tutorials/aqt/getting_started.ipynb:131]: This needs to be in `$$` signs to render correctly.

File: docs/aqt/devices.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# AQT devices
+
+AQT offers access to several quantum computing devices, so-called backends, ranging from real-hardware ion traps with various number of ions to quantum computing simulators including different noise models. To get an overview of available devices visit [www.aqt.eu](https://www.aqt.eu) and get direct access to the devices via the [AQT gateway portal](https://api-gateway.aqt.eu).
```
Comment (balopat) [inline docs/aqt/devices.md:3]: gateway-portal.aqt.eu and gateway.aqt.eu seem to work, api-gateway.aqt.eu doesn't - is this intentional?

File: docs/aqt/devices.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Tokens
+
+The AQT API to access backends uses token-based authentication. In order to be able to submit quantum circuits via quantum programming software development kits you need to supply these tokens. Once you have successfully subscribed to an AQT backend you can retrieve the token on the [AQT gateway portal](https://api-gateway.aqt.eu) and use it in your quantum programs or Jupyter notebook tutorials.
```
Comment (balopat) [inline docs/aqt/devices.md:7]: same here - gateway-portal.aqt.eu and gateway.aqt.eu seem to work, api-gateway.aqt.eu doesn't

**Review by balopat — CHANGES_REQUESTED** [review]
This looks great, thank you! I added some comments.
Also, a couple more things: 

1. we'll need to add the headers to book.yaml 
2. and to readthedocs rtd_docs/index.rst. 
3. I'd love to have Access and Authentication separated out into the Guides section that would explain the users where they can sign up and then point them to the tutorial. We'll have similar ones for Pasqal and Google as well.  Basically some of the sentences from the top section of your Getting Started guide but with a little bit more explanation what they can expect - e.g. I managed to sign up with my Google account and requested a subscription (the interface was clear), but it is unclear  when how I will get a token.
I'm happy to add 1 and 2 if you're okay with me pushing to your branch.

File: docs/tutorials/aqt/getting_started.ipynb (hunk 2 above)
Comment (alfrisch) [reply docs/tutorials/aqt/getting_started.ipynb:131]: > I'm happy to add 1 and 2 if you're okay with me pushing to your branch.

sure, go ahead, you can push to my branch

**Comment by alfrisch** [comment]
> I'm happy to add 1 and 2 if you're okay with me pushing to your branch.

sure, go ahead, you can push to my branch

File: docs/tutorials/aqt/getting_started.ipynb (hunk 2 above)
Comment (alfrisch) [reply docs/tutorials/aqt/getting_started.ipynb:131]: changed to `$$`

File: docs/aqt/devices.md (hunk 3 above)
Comment (alfrisch) [reply docs/aqt/devices.md:3]: thanks for pointing that out, updated URLs accordingly

File: docs/aqt/devices.md (hunk 4 above)
Comment (alfrisch) [reply docs/aqt/devices.md:7]: thanks for pointing that out, updated URLs accordingly

**Comment by alfrisch** [comment]
@googlebot I fixed it.

**Review by alfrisch — COMMENTED** [review]
changes have been added

**Review by balopat — APPROVED** [review]
Thank you! I added a next steps section as well that points to the getting started guide.

**Comment by balopat** [comment]
@googlebot please rescan

**Pull request merged**